src/app/concern/concern.component.html

```diff
@@ -1 +1,171 @@
-<p>concern works!</p>
+<div class="table-container" *ngIf="concernHistory$ | async as concernHistory">
```

Comment:
this seems really similar to RecommendationTableComponent. is it possible to create a generic historytable component so it can be reused on both places and just pass in the datasource, column names etc?